core/unix/os_public.h

```diff
@@ -42,6 +42,10 @@
 #ifndef _OS_PUBLIC_H_
 #define _OS_PUBLIC_H_ 1
 
+#ifdef LINUX
+# include "include/sigcontext.h"
```

Comment:
Just removing signal.h from globals_shared.h results in siginfo_t not being defined in nudgesig.c. Perhaps I should just leave it for now.